src/c++/perf_analyzer/genai-perf/genai_perf/graphs/plot_manager.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+from genai_perf.graphs.box_plot import BoxPlot
+from genai_perf.graphs.heat_map import HeatMap
+from genai_perf.graphs.scatter_plot import ScatterPlot
+from genai_perf.llm_metrics import Statistics
+
+
+class PlotManager:
+    """
+    Manage details around plots generated
+    """
+
+    def __init__(self, stats: Statistics) -> None:
+        self._stats = stats
+
+    def create_default_graphs(self):
+        bp = BoxPlot(self._stats)
+        bp.create_box_plot(
+            y_key="time_to_first_tokens",
+            graph_title="Time to First Token Analysis",
+            filename_root="ttft",
+            x_label="Time to First Token of Individual Requests (seconds)",
+        )
+        bp.create_box_plot(
+            y_key="request_latencies",
+            graph_title="Total Output Time Analysis",
```

Comment:
I think we could nix "Analysis" here to keep this consistent with the rest of the titles that are "<Metric>" or "<Metric> vs <Metric>".